### DISCUSSION TITLE
Add LSI SAS 9211-8i / Dell PERC H310 SAS HBA raid controller

### DISCUSSION THREAD

**Discussion by gimmeAll** [geerlingguy/raspberry-pi-pcie-devices#611] [General]
Both cards run on the SAS2008 chipset and should be compatible with RPI5 PCIe 2.0. I don't know enough to give the full technical rundown, but I imagine using one of these cards would be an excellent option for those wanting to make a performant and cost-effective RPI home NAS, but might need testing to see if it is compatible. 

H310: https://i.dell.com/sites/doccontent/shared-content/data-sheets/Documents/dell-perc-h310-spec-sheet.pdf
9211-8i: https://docs.broadcom.com/doc/12353333

**Comment by geerlingguy** [comment]
Unfortunately from what I've found testing various LSI cards on the CM4, all that are pre 93xx-era have some bugs that mean they'll never work on any arm64 architecture, at least according to one of the Broadcom engineers who writes the mpt3sas driver code...

**Reply by gimmeAll** [reply]
if that is the case, would there be any posibility of cards post 93xx-era like the LSI 9300-8i or 9361-8i working on the RPI/CM-4/5?

**Reply by krausedj** [reply]
Is there any insight on bugs?  At a surface level I have got a 9200-16e with 5.15.92 on a CM4 seeing drives, and making a zfs pool, but that does not mean it will actually work.  Any later kernel would panic though.  I just transitioned to _bookworm_ so I will report back after some time.
